sqlx-core/src/lib.rs

```diff
@@ -104,3 +104,6 @@ pub mod mssql;
 /// sqlx uses ahash for increased performance, at the cost of reduced DoS resistance.
 use ahash::AHashMap as HashMap;
 //type HashMap<K, V> = std::collections::HashMap<K, V, ahash::RandomState>;
+
+#[cfg(feature = "time-03")]
+extern crate time_03 as time;
```

Comment:
Can we make this conditional on `feature = "postgres"` as well? It should fix those "unused" warnings.

Reply:
Or nevermind this if you're still working on support for MSSQL/SQLite

Reply:
Honestly I just tested postgres. I haven't looked into mysql and sqlite, I just fixed the compiler warnings :smile:. From a quick search it looks like sqlite never supported `time`? https://github.com/launchbadge/sqlx/tree/master/sqlx-core/src/sqlite/types

Reply:
It definitely looks to be that way, I guess no one's ever asked for it. I guess don't worry about SQLite then?

So yeah, whatever suppresses the warning works for me.